ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,69 @@
+Issue Title
+===========
```

Comment:
I was going for simplicity, since I knew I wasn't going to have any headers deeper than 2. It was intentional, in that I wouldn't want to encourage issues or PR descriptions with more than 2 levels of headings. I believe we cover that we prefer the # notation in our contributing.md.

Thoughts from others?